roxie/ccd/ccdqueue.cpp

```diff
@@ -2097,10 +2122,7 @@ class RoxieThrottledPacketSender : public Thread
             break;
         }
         if (length > maxPacketSize)
```

Comment:
Probably not a part of this PR, but should we subtract a few bytes from maxPacketSize to allow for IP+UDP header ?

Reply:
Possibly - sounds like a new Jira though

Reply:
I agree, new JIRA